core/src/main/java/com/linecorp/armeria/client/logging/LoggingClientBuilder.java

```diff
@@ -55,4 +60,28 @@
     newDecorator() {
         return this::build;
     }
+
+    /**
+     * Sets the {@link Sampler} that determines which request needs logging.
+     */
+    public LoggingClientBuilder sampler(Sampler<? super ClientRequestContext> sampler) {
+        this.sampler = requireNonNull(sampler, "sampler");
+        return this;
+    }
+
+    /**
+     * Returns the {@link Sampler} that determines which request needs logging.
+     */
+    protected Sampler<? super ClientRequestContext> sampler() {
```

Comment:
Maybe we do not need this method anymore, because `LoggingClientBuilder` can access it directly.